packages/block-library/src/term-description/editor.scss

```diff
@@ -0,0 +1,11 @@
+.wp-block-term-description__placeholder {
```

Comment:
I like the approach with the placeholder styles. Just check to have the alignments from attributes and remove the `italic`.

Other changes like `color` etc are shown in preview.